doc/applications.md

```diff
@@ -0,0 +1,151 @@
+# Creating applications
+
+RxJS is a great tool to keep your code less error prone. It does that by using pure and stateless functions. But applications are stateful, so how do we bridge the stateless world of RxJS with the stateful world of our applications?
+
+### Vanilla JS
+```js
+const button = document.querySelector('button');
+Rx.Observable.fromEvent(button, 'click')
+  .scan(count => count + 1, 0)
+  .subscribe(count => document.querySelector('#count').innerHTML = count);
+```
```

Comment:
FWIW: when I code rx chains, especially in hot code paths that I think other developers will look at, I tend to add comments above each operation like:

```js
// get a stream of click events from your button
Rx.Observable.fromEvent(button, 'click')
  // scan (reduce) to a stream of counts
  .scan(count => count + 1, 0)
  // SIDE EFFECT: render the count out to the #count element
  .subscribe(count => document.querySelector('#count').innerHTML = count);
```